pygmt/helpers/utils.py

```diff
@@ -170,7 +170,7 @@ def is_nonstr_iter(value):
 
     """
     try:
-        [item for item in value]  # pylint: disable=pointless-statement
```

Comment:
It's possible to simplify the function using this python code
```python
from collections.abc import Iterable

return isinstance(value, Iterable) and not isinstance(value, str)
```
See https://stackoverflow.com/a/61139278

Reply:
Ok, done!